test/test_gnmi_configdb_patch.py

```diff
@@ -2170,6 +2170,144 @@ def create_checkpoint(file_name, text):
     }
 ]
 
+test_data_k8s_config_patch = [
+    {
+        "test_name": "K8SEMPTYTOHALFPATCH",
+        "operations": [
+            {
+                "op": "update",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER",
+                "value": {"SERVER": {}}
+            }
+        ],
+        "origin_json": {
+            "KUBERNETES_MASTER": {}
+        },
+        "target_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {}
+            }
+        }
+    },
+    {
+        "test_name": "K8SHALFTOFULLPATCH",
+        "operations": [
+            {
+                "op": "update",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER/SERVER/disable",
+                "value": "false"
+            },
+            {
+                "op": "update",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER/SERVER/ip",
+                "value": "k8svip.ap.gbl"
+            }
+        ],
+        "origin_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {
+                    "disable": "true",
+                    "ip": ""
+                }
+            }
+        },
+        "target_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {
+                    "disable": "false",
+                    "ip": "k8svip.ap.gbl"
+                }
+            }
+        }
+    },
+    {
+        "test_name": "K8SFULLTOHALFPATCH",
+        "operations": [
+            {
+                "op": "del",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER/SERVER/disable"
+            },
+            {
+                "op": "del",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER/SERVER/ip"
+            }
+        ],
+        "origin_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {
+                    "disable": "true",
+                    "ip": ""
+                }
+            }
+        },
+        "target_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {}
+            }
+        }
+    },
+    {
+        "test_name": "K8SHALFTOEMPTYPATCH",
+        "operations": [
+            {
+                "op": "del",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER"
+            }
+        ],
+        "origin_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {
+                    "disable": "true",
+                    "ip": ""
+                }
+            }
+        },
+        "target_json": {}
+    },
+    {
+        "test_name": "K8SHALFTOEMPTYPATCH",
+        "operations": [
+            {
+                "op": "del",
+                "path": "/sonic-db:CONFIG_DB/localhost/KUBERNETES_MASTER"
+            }
+        ],
+        "origin_json": {
+            "KUBERNETES_MASTER": {
+                "SERVER": {
+                    "disable": "true",
+                    "ip": ""
```

Comment:
need to update the case. `"ip": ""` will report error with GCU due to fail yang model validation

Reply:
this unit test will not invoke GCU, and we only need to verify patch is valid.